### PR TITLE
Updated themebased text color test for restore mnemonic

### LINKED ISSUE #828: `e2e` - flaky test on mnemonic tests
#### What would you like to be added:
https://dashboard.cypress.io/projects/oqk3fk/runs/2085/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&utm_source=github

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:
e2e - flaky test on mnemonic tests
#### Which issue(s) does this PR fixes?:

Fixes #828

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: cypress/support/onboardingCommands.ts [hunk 1]
```diff
@@ -7,34 +7,34 @@ declare global {
        * @description Start onboarding mnemonic wallet process
        * @param {string[]} recoveryWords - pass an empty array to be filled later
        */
```
Comment (thedoublejay) [inline cypress/support/onboardingCommands.ts:9]: why is this change needed? @fullstackninja864

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: cypress/support/onboardingCommands.ts (hunk 1 above)
Comment (fullstackninja864) [reply cypress/support/onboardingCommands.ts:9]: These are just lint changes, which was throwing error/warning as we have added lint for test cases too

Comment (fullstackninja864) [reply cypress/support/onboardingCommands.ts:9]: In order to support the dark and light mode i had updated this file and then after before creating commit I have to fix them to pass pre-commit hooks.

**Review by thedoublejay — APPROVED** [review]
